modules/nf-core/shapeit5/ligate/main.nf

```diff
@@ -0,0 +1,38 @@
+process SHAPEIT5_LIGATE {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda "bioconda::shapeit5=1.0.0"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/shapeit5:1.0.0--h0c8ee15_0':
+        'quay.io/biocontainers/shapeit5:1.0.0--h0c8ee15_0'}"
+
+    input:
+    tuple val(meta), path(input_list, stageAs: 'dir*/*'), path (input_list_index, stageAs: 'dir*/*')
```

Comment:
some people don't like the `stageAs`. Don't count me as one of them, just saying that you can get some comments on it later (they usually prefer that all input files have different names)

Reply:
I understand completely I will do another way!